### ISSUE TITLE
REPL output despite  `OUTPUTLOG=0`

### ISSUE THREAD

**Issue by jjstickel** [jump-dev/Xpress.jl#101]
With JuMP 0.21.4 and Xpress v0.11.1, I get some extra output in the REPL, even with `OUTPUTLOG=0`. Example:
```
using JuMP
import Xpress

model = Model(with_optimizer(Xpress.Optimizer, OUTPUTLOG=0))

@variable(model, 0 <= x <= 2 )
@variable(model, 0 <= y <= 30 )
@objective(model, Max, 5x + 3y )
@constraint(model, 1x + 5y <= 3.0 )

optimize!(model)
println("Objective value: ", objective_value(model))
```
produces:
```
Reading Problem
Problem Statistics
           0 (      0 spare) rows
           0 (      0 spare) structural columns
           0 (      0 spare) non-zero elements
Global Statistics
           0 entities        0 sets        0 set members
Objective value: 10.600000000000001
```

**Comment by kdheepak** [comment]
Can you try the following?

```julia
using JuMP
import Xpress
OPTIMIZER = Xpress.Optimizer();
@show MOI.get(OPTIMIZER, MOI.RawParameter("OUTPUTLOG"))
@show MOI.get(OPTIMIZER, MOI.RawParameter("logfile"))
```

**Comment by jjstickel** [comment]
Here are the last two lines of the above:
```
julia> @show MOI.get(OPTIMIZER, MOI.RawParameter("OUTPUTLOG"))
MOI.get(OPTIMIZER, MOI.RawParameter("OUTPUTLOG")) = 0
0

julia> @show MOI.get(OPTIMIZER, MOI.RawParameter("logfile"))
MOI.get(OPTIMIZER, MOI.RawParameter("logfile")) = ""
""
```

**Comment by rafabench** [comment]
The keyword `with_optimizer` starts the solver and then sets the parameters (JuMP default). That's why you've got extra output in the REPL. You should build the model as the following, because this way it sets the parameters before starting the solver.
`model = Model(() -> Xpress.Optimizer(OUTPUTLOG=0))`

**Comment by kdheepak** [comment]
Thanks for commenting @rafabench, I was just going to suggest the same. This does what a user would expect:

![Screen Shot 2020-10-27 at 5 55 05 PM](https://user-images.githubusercontent.com/1813121/97374266-8cf86600-187d-11eb-8e61-a171ab68021f.png)

**Comment by jjstickel** [comment]
Thanks for the workaround suggestions. I am using JuMP via InfrastructureModels.jl, the internals of which create the model and then adds the optimizer later (allowing any optimizer, not just Xpress). It does not look like the workarounds will work for this case. But #103 will resolve this issue?

**Comment by jjstickel** [comment]
I found a workaround that works when setting the Xpress optimizer after initial creation of the model:
```
model = Model()
xpress = () -> Xpress.Optimizer(OUTPUTLOG=0)
set_optimizer(model, xpress)
```
But it will be nice to have the fix in #103. Thanks again for responses.
